src/WebJobs.Script/Workers/Http/HttpFunctionInvocationDispatcher.cs

```diff
@@ -59,7 +58,7 @@ internal class HttpFunctionInvocationDispatcher : IFunctionInvocationDispatcher
 
         public FunctionInvocationDispatcherState State { get; private set; }
 
-        internal Task InitializeJobhostLanguageWorkerChannelAsync(int attemptCount)
+        internal Task InitializeHttpeWorkerChannelAsync(int attemptCount)
```

Comment:
nit - should this be "InitializeHttpWorkerChannelAsync" (without the "e" after Http)?

Reply:
thanks for catching typo!